Airports/EGPG/SMR/geo.txt

```diff
@@ -0,0 +1,153 @@
+EGPG Cumbernauld               S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
+; - 0_Inner
+N055.58.25.048 W003.58.53.333 N055.58.25.009 W003.58.53.312 smrWhite
+N055.58.25.009 W003.58.53.312 N055.58.24.955 W003.58.53.632 smrWhite
+N055.58.24.955 W003.58.53.632 N055.58.24.992 W003.58.53.651 smrWhite
+N055.58.24.992 W003.58.53.651 N055.58.25.048 W003.58.53.333 smrWhite
+; - 0_Outer
+N055.58.25.095 W003.58.53.251 N055.58.24.997 W003.58.53.201 smrWhite
+N055.58.24.997 W003.58.53.201 N055.58.24.904 W003.58.53.714 smrWhite
+N055.58.24.904 W003.58.53.714 N055.58.25.001 W003.58.53.765 smrWhite
+N055.58.25.001 W003.58.53.765 N055.58.25.095 W003.58.53.251 smrWhite
+; - 7
+N055.58.24.939 W003.58.53.174 N055.58.24.835 W003.58.53.112 smrWhite
+N055.58.24.835 W003.58.53.112 N055.58.24.820 W003.58.53.656 smrWhite
+N055.58.24.820 W003.58.53.656 N055.58.24.849 W003.58.53.671 smrWhite
+N055.58.24.849 W003.58.53.671 N055.58.24.862 W003.58.53.228 smrWhite
+N055.58.24.862 W003.58.53.228 N055.58.24.921 W003.58.53.261 smrWhite
+N055.58.24.921 W003.58.53.261 N055.58.24.939 W003.58.53.174 smrWhite
+; - CL
```

Comment:
Could you name everything so that if it needs to be edited later its easy to find?  If this was for taxiway A, you might want to do this:
```suggestion
;Taxiway_A
```

Reply:
> 
> 
> Could you name everything so that if it needs to be edited later its easy to find? If this was for taxiway A, you might want to do this:

Didn't mean to resolve this one, CL is the centerline